index.js

```diff
@@ -580,6 +593,150 @@ module.exports = {
 
     self.rewriteUrl = url => {
       return url;
-    }
+    };
+
+    self.getPagesTree = async (req) => {
```

Comment:
Maybe I should have checked if the locale is private?

Reply:
No, it's up to the developer to decide when to use the method and for what locales.